pymongo/collection.py

```diff
@@ -2553,7 +2553,7 @@ def watch(
             pipeline stages are valid after a ``$changeStream`` stage, see the
             MongoDB documentation on change streams for the supported stages.
           - `full_document` (optional): The fullDocument to pass as an option
-            to the ``$changeStream`` stage. Allowed values: 'updateLookup'.
+            to the ``$changeStream`` stage. Allowed values: 'updateLookup', 'whenAvailable', 'required'.
```

Comment:
Can you wrap these lines to at most our 100 char limit? Same for full_document_before_change.